app/views/posts/index.html.erb

```diff
@@ -1 +1,33 @@
-<h1>Here is a list of posts for a given user<h1>
\ No newline at end of file
+<h1>Blog App</h1>
+<div class='user-content'>
+  <%= image_tag @user.photo, alt: "#{@user.name} profile image", class: 'profile'%>
+ <div class='user-content2'>
+   <%= link_to @user,class:'user-content-details' do %>
+   <h2><%= @user.name %></h2>
+   <p class='user-post-total'>Number of posts: <%=@user.post_counter%> </p>
+   <%end%>
+ </div>
+</div>
+<ul>
+<% if @posts.empty? %>
+  <p class='no-posts'>No posts</p>
+<% else %>
+<% @posts.each do |post|%>
+<li>
+<%=link_to user_post_path(post.author, post), class:'post-content' do%>
+  <div class='post_subcont'>
+  <h3><%=post.title%></h3>
+  <div><%=post.text[0, 100]%><%='...' if post.text[51]%></div>
+  <p class='post-total'>Comments: <%=post.comment_counter%>, Likes: <%=post.likes_counter%></p>
+  </div>
+  <%end%>
+  <ul class='comment-content'>
+  <% comments.each do |comment|%>
```

Comment:
- [ ] Great job on your progress! 👏 I encountered the following error when clicking on the "See all posts" button/link ⚠. Please resolve this issue. Additionally, please ensure that you render the five most recent comments on the `app/views/posts/index.html.erb` page using the method defined in the model. 💪


![image](https://github.com/emmanuelsteven/MyBlogApp/assets/106119105/2aad70ea-16e9-4e76-92b2-54f615ceb22d)